tests/base.py

```diff
@@ -155,6 +156,16 @@ def setUp(self):
             )
             response_2.save()
             response_2.missing_resources.append(resource)
+            structure = Structure(
+                web_id='gsgsggg',
```

Comment:
Hello @kwanj-k , I suggest that we use a more descriptive `web_id` here. You can get a sample of a web id from the front-end and we use that.

Reply:
@nakatuddesuzan the id comes with the format ``` id: "b05fc5f2-b4aa-4f48-a8fb-30bdcc3fc968"  ``` 
I decided to call it web_id as we have sqlachemy id